pkg/yurtctl/cmd/yurtinit/phases/install_yurt_addons.go

```diff
@@ -77,7 +77,7 @@ func runInstallYurtAddons(c workflow.RunData) error {
 	if err := kubeutil.DeployYurttunnelServer(client, nil, fmt.Sprintf("%s/%s:%s", imageRegistry, YurtContants.YurtTunnelServer, version), runtime.GOARCH); err != nil {
 		return err
 	}
-	if err := kubeutil.DeployYurttunnelAgent(client, nil, fmt.Sprintf("%s/%s:%s", imageRegistry, YurtContants.YurtTunnelAgent, version)); err != nil {
+	if err := kubeutil.DeployYurttunnelAgent(client, "", fmt.Sprintf("%s/%s:%s", imageRegistry, YurtContants.YurtTunnelAgent, version)); err != nil {
```

Comment:
maybe it's more friendly to add `--tunnelserver-addr` parameter for `yurtctl init` command.

Reply:
done